test/desktop-lite/test_vnc_resolution_as_container_env_var.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+# Optional: Import test library
+source dev-container-features-test-lib
+
+# Check if xtigervnc & noVnc processes are running after successful installation and initialization
+check_process_running() {
+    port=$1
+    # Get process id of process running on specific port
+    PID=$(lsof -i :$port | awk 'NR==2 {print $2}')
+    GREEN='\033[0;32m'; NC='\033[0m'; RED='\033[0;31m'; YELLOW='\033[0;33m';
+    if [ -n "$PID" ]; then
+        CMD=$(ps -p $PID -o cmd --no-headers)
+        echo -e "${GREEN}Command running on port $port: ${YELLOW}$CMD${NC}"
+    else
+        echo -e "${RED}No process found listening on port $port.${NC}"
+        exit 1
+    fi
+}
```

Comment:
Is it possible to add a check to validate `VNC_RESOLUTION` variable value to matches with what we passed?

Reply:
added